### PR TITLE
New Cards

### PR DESCRIPTION
**Pull request by SaraVieira**
**What kind of change does this PR introduce?**

**What is the current behavior?**

**What is the new behavior?**

**Checklist**:

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged 
- [ ] Added myself to contributors table

### CODE REVIEW

**Comment by CompuIves** [comment]
Niiiice, looks really good already! I have some very small nitpicks:

- On hover I would show the dark overlay anyway, no matter if there's no description or not. This lets the user know that there is the possibility of the description.
- I would also show the dark overlay when you hover the card itself (under). This might need some JS magic though, less nice but better UX.
- The icons of the stats can maybe be a bit darker, inspiration: https://twitter.com/steveschoger/status/872865304719892480
- There is a jump when switching pages, that's probably because the loading of the images cause the card height to jump

After having these things resolved we're ready to rumble!

**Comment by SaraVieira** [comment]
Everything fixed, maybe?

**Comment by CompuIves** [comment]
Nice!!! One last tiiiiiiny thing I forgot to add! Could you add `M` (million) support? 

image

File: packages/common/src/components/SandboxCard/index.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  small?: boolean;
+  noHeight?: boolean;
+  defaultHeight?: number;
+  noMargin?: boolean;
+  selectSandbox: (params: Sandbox) => void;
+};
+
+const kFormatter = (num: number): number | string => {
+  if (num > 9999) {
+    return (num / 10000).toFixed(1) + 'M';
```
Comment (CompuIves) [inline packages/common/src/components/SandboxCard/index.tsx]:
```suggestion
    return (num / 1000000).toFixed(1) + 'M';
```

**Pull request merged**